lua/compe_treesitter/init.lua

```diff
@@ -0,0 +1,87 @@
+local ts_exists = pcall(require, "nvim-treesitter")
```

Comment:
I think this is not enough because the still require `nvim-treesitter.locals` without pcall.

Reply:
I think if someone have nvim-treesitter installed, they will always have `nvim-treesitter.locals` so I assumed we only need to check the top module.

Reply:
If the user does not have `nvim-treesitter`, `require'nvim-treesitter.locals` will throw exception.

1. `local ts_exists = pcall(require, "nvim-treesitter")` ignore exception.
2. `local ts_locals = require('nvim-treesitter.locals')`

Currently, nvim-compe uses `pcall` for public apis.

Could you check with `let g:compe.debug = v:true`? (If you have a time)

Reply:
ah, yeah, that makes sense. Will fix it in a moment

Reply:
Good! Thank you!